pkg/resource/scalable_target/custom_api.go

```diff
@@ -40,3 +40,13 @@ func (rm *resourceManager) customSetLastModifiedTime(ko *svcapitypes.ScalableTar
 	currentTime := metav1.Time{Time: time.Now().UTC()}
 	ko.Status.LastModifiedTime = &currentTime
 }
+
+// customCheckRequiredFieldsMissingMethod returns true if there are any fields
+// for the ReadOne Input shape that are required but not present in the
+// resource's Spec or Status
+func (rm *resourceManager) customCheckRequiredFieldsMissingMethod(
```

Comment:
nit: Remove `Method` from the method name. We know it's a method :)

Reply:
done